docs/hoon/library/1a.md

```diff
@@ -25,23 +25,37 @@ An atom.
 Source
 ------
 
-    ++  add                                                 ::  add
+    ++  add                                                
       ~/  %add
-      |=  [a/@ b/@]
+      |=  [a=@ b=@]
       ^-  @
       ?:  =(0 a)  b
       $(a (dec a), b +(b))
-
+::
 Examples
 --------
 
     > (add 2 2)
     4
+
     > (add 1 1.000.000)
     1.000.001
+
     > (add 1.333 (mul 2 2))
     1.337
 
+    > `@`'a'                                     ::`'a'` is an atom represented as a different data type.
+    97                                             We cast it to the aura `@` to see what its integer representation is.
+    > (add 'a' 4)                                
+    101
+    (add 'a' 'b')
+    195  
```

Comment:
Don't include the cord addition example.  This isn't a normal use case for the gate.

The hex addition is nice though.